# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../firefox/startpage.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- mozilla/browser/base/content/browserconfig.properties.orig	2005-02-22 08:33:46.014560240 +0100
+++ mozilla/browser/base/content/browserconfig.properties	2005-02-22 08:34:18.282654744 +0100
@@ -1,3 +1,3 @@
 # Do NOT localize or otherwise change these values
-browser.startup.homepage=http://www.mozilla.org/products/firefox/central.html
-browser.startup.homepage_reset=http://www.mozilla.org/products/firefox/central.html
\ No newline at end of file
+browser.startup.homepage=http://www.t2-project.org/startpage/
+browser.startup.homepage_reset=http://www.t2-project.org/startpage/
